### DISCUSSION TITLE
Backrest was keep on failing while trying to upload 240GB of files. Resume worked and now it stopped resuming due to LOCK

### DISCUSSION THREAD

**Discussion by CodeWithCJ** [garethgeorge/backrest#613] [Q&A] [answered]
It stopped couple of times. Re-running backup again resumed. Now it stopped due to some LOCK. Can this be resumed as the backup didn't complete fully. Or Do I need to delete and start from the scratch?  Should I click on UNLOCK REPO and re-try? I am worried if my backup might be in corrupted state or good to unlock and continue.

**Comment by garethgeorge** [comment, marked as answer]
It's not the best that you're getting frequent timeouts (enough so that restic actually gives up after retrying). That looks like an issue with your network stability that's preventing a reliable upload.

That said, if you can retry enough to get a successful backup and then run a "check" operation that comes back OK then you can be confident that all of your data is uploaded correctly. 

unlock is always safe if you know you don't have other installs backing up to the same location. It's a feature that is supported for shared restic repositories with multiple backup clients. In most cases, the safest setup is one repository per backup client with `Auto Unlock` enabled in the repo settings.

**Reply by CodeWithCJ** [reply]
Thank you. Prone and Check executed successfully.
